pkg/controller/form.go

```diff
@@ -34,8 +34,5 @@ func BindForm(w http.ResponseWriter, r *http.Request, data interface{}) error {
 	// don't need to be captured. By default schema decoder is very struct.
 	decoder.IgnoreUnknownKeys(true)
 
-	if err := decoder.Decode(data, r.PostForm); err != nil {
-		return err
```

Comment:
We should actually probably wrap this error in some meaningful context

Reply:
well probably not part of this pr :)

Reply:
Then leave it as-is and do it as a follow-up?

Reply:
Not sure why? The code is idempotent as is, but 4 times more readable (4:1 line ratio )